### PR TITLE
Rename namespaces for func_test_utils

### PR DESCRIPTION
**Pull request by olpipi**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/auto_batch/tests/functional/behavior/plugin/auto_batching_tests.hpp [hunk 1]
```diff
@@ -91,7 +91,7 @@ class AutoBatching_Test : public BehaviorTestsUtils::IEPluginTestBase,
 
                 std::vector<std::vector<uint8_t>> inData;
                 for (auto n : inputs) {
-                    auto blob = FuncTestUtils::createAndFillBlob(n.second->getTensorDesc());
+                    auto blob = ov::test::utils::createAndFillBlob(n.second->getTensorDesc());
```
Comment (iefode) [inline src/plugins/auto_batch/tests/functional/behavior/plugin/auto_batching_tests.hpp:94]: need to be moved to OV2.0

File: src/tests/test_utils/functional_test_utils/include/functional_test_utils/legacy/blob_utils.hpp [hunk 2]
```diff
@@ -309,7 +310,7 @@ compareBlobs(const InferenceEngine::Blob::Ptr &res, const InferenceEngine::Blob:
```
Comment (iefode) [inline src/tests/test_utils/functional_test_utils/include/functional_test_utils/legacy/blob_utils.hpp:310]: Let's start to move to OV2.0

File: src/tests/test_utils/functional_test_utils/include/functional_test_utils/precision_utils.hpp [hunk 3]
```diff
@@ -6,51 +6,51 @@
...
 
-namespace FuncTestUtils {
-namespace PrecisionUtils {
+namespace ov {
+namespace test {
+namespace utils {
 
 // Copied from inference-engine/src/inference_engine/src/ie_ngraph_utils.hpp
-inline ::ngraph::element::Type convertIE2nGraphPrc(const InferenceEngine::Precision& precision) {
+inline ::ov::element::Type convertIe2OvPrc(const InferenceEngine::Precision& precision) {
```
Comment (iefode) [inline src/tests/test_utils/functional_test_utils/include/functional_test_utils/precision_utils.hpp:17]: To remove?

File: src/tests/test_utils/functional_test_utils/src/test_model/test_model.cpp [hunk 4]
```diff
@@ -15,8 +15,9 @@
 #include "openvino/pass/serialize.hpp"
 #include "ie_ngraph_utils.hpp"
 
-namespace FuncTestUtils {
-namespace TestModel {
+namespace ov {
+namespace test {
+namespace utils {
```
Comment (iefode) [inline src/tests/test_utils/functional_test_utils/src/test_model/test_model.cpp:21]: to remove this file and replace function using

**Comment by olpipi** [comment]
Hi @iefode 
This PR is outdated. Let me please rework it.

**Pull request closed without merging**